mirgecom/steppers.py

```diff
@@ -76,7 +76,7 @@ def _is_unevaluated(actx, ary):
 def _advance_state_stepper_func(rhs, timestepper, state, t_final, dt=0,
                                 t=0.0, istep=0, pre_step_callback=None,
                                 post_step_callback=None, force_eval=None,
-                                local_dt=False, max_steps=None):
+                                local_dt=False, max_steps=None, compile_rhs=None):
```

Comment:
Should this be `False` by default (same for leap)?

Reply:
Strictly speaking the default value doesn't really matter for these, since they're only called from `advance_state` and it always passes its own value for `compile_rhs`. I couldn't tell if the convention was to default to `None` (like `force_eval`) or `True`/`False` (like `local_dt`) so I just picked one. I can switch to the other if that would be preferable.

Reply:
mypy would likely complain if you add a `:bool` annotation, so I'd just go ahead and make it an actual bool instead.